cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRMutationDataReader.java

```diff
@@ -137,7 +137,7 @@ private void loadMutationRecordsFromJson(CVRData cvrData) {
         }
         log.info("Loaded " + String.valueOf(recordsToAnnotate.size()) + " records from JSON");
         try {
-            annotateRecordsWithPOST(recordsToAnnotate);
+            annotateRecordsWithPOST(recordsToAnnotate, true);
```

Comment:
https://github.com/knowledgesystems/cmo-pipelines/pull/652/files#diff-7258385239600627f9ec5efeef299ee3L146

Reply:
Added this argument `reannotate` back in to differentiate between whether we need to "force annotation".

For records read from JSON we want to ALWAYS annotate everything. This original logic was lost -- refer to link above to see what was dropped.